base/compiler/optimize.jl

```diff
@@ -33,6 +33,10 @@ const IR_FLAG_NOTHROW     = 0x01 << 5
 
 const TOP_TUPLE = GlobalRef(Core, :tuple)
 
+const MAX_INLINE_COST = typemax(UInt16)
+is_inlineable(src::Union{CodeInfo, Vector{UInt8}}) = ccall(:jl_ir_inlining_cost, UInt16, (Any,), src) != MAX_INLINE_COST
+set_inlineable!(src::CodeInfo, val::Bool) = src.inlining_cost = val ? 0x0010 : MAX_INLINE_COST
```

Comment:
Convert `0x0010` to a constant so that it has a clear meaning?

Reply:
```suggestion
set_inlineable!(src::CodeInfo, val::Bool) = src.inlining_cost = (val ? 0x0010 : MAX_INLINE_COST)
```
style nit: hint to the reader the assignment is a ternary expression to avoid garden-path reading